### ISSUE TITLE
stylelint: Could not find "stylelint-config-sass-guidelines". Do you need a `configBasedir`?

### ISSUE THREAD

**Issue by ThePeach** [shinnn/vscode-stylelint#37]
This is very much the same error that was raised in #25 but unfortunately I do have the extended config ("stylelint-config-sass-guidelines") already installed:

```
$ ls -ld node_modules/stylelint-config-sass-guidelines/
drwxr-xr-x 2 peach peach 4096 Oct 21 12:01 node_modules/stylelint-config-sass-guidelines/
```

I do have no problems in running it manually, either using stylelint executable or gulp.

**Comment by shinnn** [comment]
I just tried to use `stylelint-config-sass-guidelines` via vscode-stylelint v0.18.0, and worked perfectly.

**Comment by ThePeach** [comment]
@shinnn could it be a problem with the project residing in a subdirectory?

**Comment by shinnn** [comment]
@ThePeach Can you show me the directory tree of your project that includes at least the paths of your stylelint configuration file, `node_modules` and the CSS file you're trying to lint?

**Comment by ThePeach** [comment]
the relevant situation is the following:

    ~/project/
      - node_modules/
      - styleguide/
        - node_modules/
        - source/css/
          - dir/
            - objects.scss
          - style.scss
        - .stylelintrc
        - gulpfile.js
        - package.json
      - package.json

the second-level `package.json` is the one I'm currently using with all the dev deps related to stylelint, as follows:

    {
        "devDependencies": {
            "gulp-stylelint": "^3.4.0",
            "stylelint": "^7.5.0",
            "stylelint-config-sass-guidelines": "^1.1.0",
            "stylelint-scss": "^1.3.4",
            "stylelint-selector-bem-pattern": "^1.0.0"
        }
    } 

although the gulp one is particularly irrelevant in this context.

**Comment by lexder** [comment]
@ThePeach came here after i had opened my "work" folder containing multiple projects, vscode-stylelint wasn't working; opening it separately made the plugin work

**Comment by shinnn** [comment]
This bug is caused by [this line](https://github.com/stylelint/stylelint/blob/ef82477f24706dd1a1479db3751b749c5445e2d7/src/standalone.js#L59) of stylelint. I'm sure that it should receive the second argument like [this line](https://github.com/stylelint/stylelint/blob/ef82477f24706dd1a1479db3751b749c5445e2d7/src/standalone.js#L78). I'll open an issue on https://github.com/stylelint/stylelint/issues when I have time.

**Comment by kumarharsh** [comment]
This is a big problem for me as my `ui/` folder has the css, which is 1 level deep from my workspaceRoot. This plugin is apparently not reading my `ui/.stylelintrc.json` file, and if I try to put the config into the `.vscode/settings.json` file in my workspaceRoot, then this error is thrown.

**Comment by shinnn** [comment]
Will be fixed in https://github.com/stylelint/stylelint/pull/2516.
